src/request_handler.rb

```diff
@@ -0,0 +1,66 @@
+require 'yaml'
+
+class RequestHandler
+  attr_reader :message, :bot, :receiver
+
+  def initialize(receiver)
+    @receiver = receiver
+    @message = receiver.message
+    @bot = receiver.bot
+
+    @answers = YAML.load_file(File.dirname(__FILE__) + '/data/messages.yml')
+
+    handle_message
+  end
+
+  def handle_message
+    case message.text
+      when 'What time is it?'
+        send_response Time.now.strftime("%T")
+      when '/start'
+        send_response @answers.fetch('start')
```

Comment:
If you're using `.fetch`, use default value for hash lookup or add rescue block, otherwise it's better to use good old `.[]` method, ’cause it's [much faster](https://github.com/JuanitoFatas/fast-ruby#hash-vs-hashfetch-code).